src/shared/lib/StoreUtils.ts

```diff
@@ -55,12 +57,21 @@ import {AlterationTypeConstants} from "../../pages/resultsView/ResultsViewPageSt
 import {stringListToIndexSet} from "public-lib/lib/StringUtils";
 import {GeneticTrackDatum_Data} from "../components/oncoprint/Oncoprint";
 import {normalizeMutations} from "../components/mutationMapper/MutationMapperUtils";
+import client from 'shared/api/cbioportalClientInstance';
 
 export const ONCOKB_DEFAULT: IOncoKbData = {
     uniqueSampleKeyToTumorType : {},
     indicatorMap : {}
 };
 
+export const CompleteProfileTypeSignature = {
+    UNKNOWN: undefined,
+    WHOLE_EXOME_SEQ: 'WES',
+    WHOLE_GENOME_SEQ: 'WGS'
+};
+
+export const noGenePanelUsed:(o:string|undefined) => boolean = (o) => o === CompleteProfileTypeSignature.UNKNOWN || o === CompleteProfileTypeSignature.WHOLE_EXOME_SEQ || o === CompleteProfileTypeSignature.WHOLE_GENOME_SEQ;
```

Comment:
A few things about this

- I think just defining a function is easier to read than defining a constant with a function type. So just `export function noGenePanelUsed(...):boolean {` (the return type is probably unnecessary IMO since it's easy to see by inspection, but up to you)
- It would help a lot in understanding what this is to name the parameter something more descriptive than `o` - `genePanelId` I think?